docs/faq.rst

```diff
@@ -393,6 +393,61 @@ curve for beginners by making it easy to get started and easy to build up to
 more advanced projects.
 
 
+What's the difference between when_pressed, is_pressed and wait_for_press?
+==========================================================================
```

Comment:
Niiiiiiice!

Reply:
Definitely needed - but hard to capture the "question" - it's usually people saying "my code doesn't work" and they've just done something like reassigned `when_pressed` in a loop - at least we'll have something to link to. Also hard to cover the general case but it should help.